mita/openstack.py

```diff
@@ -238,50 +237,36 @@ def _get_network(self, network: Optional[str] = None) -> OpenStackNetwork:
 
     def _create_vm_node(self) -> None:
         """Create the instance using the provided data."""
-        try:
-            logger.info("Instantiating VM with name %s", self.node_name)
-            self.node = self.driver_v2.create_node(
-                name=self.node_name,
-                image=self._get_image(),
-                size=self._get_flavor(),
-                ex_userdata=self.cloud_data,
-                networks=[self._get_network(self.vm_network)],
-            )
-
-            if self.node is None:
-                raise NodeErrorState(
-                    "Unable to create the instance {}".format(self.node_name)
-                )
-
-            logger.info("%s is created", self.node.name)
-            return
-        except SSLError:
-            logger.error("Connection failed, probably a timeout was reached")
-        except BaseException as be:  # noqa
-            logger.error(be)
+        logger.info("Starting to create VM with name %s", self.node_name)
+
+        self.node = self.driver_v2.create_node(
+            name=self.node_name,
+            image=self._get_image(),
+            size=self._get_flavor(),
+            ex_userdata=self.cloud_data,
+            networks=[self._get_network(self.vm_network)],
+        )
 
-        raise NodeErrorState("Failed to create the instance {}".format(self.node_name))
+        if self.node is None:
+            raise NodeErrorState(f"Failed to create {self.node_name}")
 
     def _wait_until_vm_state_running(self):
         """Wait till the VM moves to running state."""
         timeout = datetime.timedelta(seconds=600)
         start_time = datetime.datetime.now()
-        while True:
-            logger.info("Waiting for %s state to be running ", self.node_name)
```

Comment:
I kinda liked this message, but it was noisy. I wonder if it would be more useful to log the total time that we've waited so far, like "Waited {timedelta} for {self.node_name} to start running" or something. What do you think?

Reply:
Finding the time taken to move to `running` state is straight forward. I agree it is a useful data point if monitor...

Reply:
Oh, I was thinking more like logging it while we're waiting, before `node.state == "running"`, so the user has some visual indication of progress. But this implementation is fine too.